### ISSUE TITLE
Support for Continuity Camera

### ISSUE THREAD

**Issue by habovh** [Gaulomatic/AirPodsSanity#11]
Hi @Gaulomatic, it's me again 😅

For some reason Continuity Camera does not appear in the list of "Which are AirPods?" devices on AirPodsSanity. This is likely because they are most likely "temporary" devices.

![Screenshot 2024-05-29 at 11 07 18](https://github.com/Gaulomatic/AirPodsSanity/assets/4186230/1e291ad9-f0a0-42c6-8dc5-bcf099995aa3)

Selecting "All speakers" does not work either when using Continuity Camera, so as soon as I'm using my iPhone as a webcam, the audio input switches to my iPhone as well.

Would that be something AirPodsSanity would be able to handle?

**Comment by Gaulomatic** [comment]
I don't think that Continuity provides output capabilities, thus this currently doesn't work. It's because the app is determining AirPods by output devices. If I really think about it, this only works because their output name equals the input name. And since the app is basically an input device swap thingy, this doesn't even make any sense :-)

Please test it with this test build. Here you can select input devices as "AirPods", so I assume this should work for your case. Maybe. :-)

[AirPods Sanity 1.0.5.0 Beta.dmg.zip](https://github.com/Gaulomatic/AirPodsSanity/files/15486109/AirPods.Sanity.1.0.5.0.Beta.dmg.zip)

**Comment by habovh** [comment]
Ah yes, I guess that would make sense to actually base the decision on the input device instead! Didn't think much about that either.

It seems this beta version works for this use case, I am able to select the iPhone from Continuity and it seems to override to my preferred input device as expected now.
It seems however that input devices that are designated as "AirPods" but are not available later are no longer shown in the list at all. Once the device is re-connected to the Mac it appears again and it remembers that it was an "AirPods" device. It would be nice to have a way to manage "offline devices that were considered as AirPods", just so we can be sure that once they reconnect they'll behave as expected.

Thanks for the beta build! ⚡
